optimize/self-managed/optimize-deployment/configuration/system-configuration.md

```diff
@@ -181,6 +181,64 @@ Define a secured connection to be able to communicate with a secured Elasticsear
+| ------------------------------------------------ | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| opensearch.settings.index.prefix                 | optimize      | The prefix prepended to all Optimize index and alias namopensearch. Custom values allow to operate multiple isolated Optimize instances on one Opensearch cluster. <br /><br />NOTE: Changing this after Optimize was already run before will create new empty indexes.                                                                                                                                                |
+| opensearch.settings.index.number_of_replicas     | 1             | How often data should be replicated to handle node failures.                                                                                                                                                                                                                                                                                                                                                           |
+| opensearch.settings.index.number_of_shards       | 1             | How many shards should be used in the cluster for process instance and decision instance indices. All other indices will be made up of a single shard. <br /><br />Note: this property only applies the first time Optimize is started and the schema/mapping is deployed on Opensearch. If you want this property to take effect again, you need to delete all indices (and with that all data) and restart Optimize. |
+| opensearch.settings.index.refresh_interval       | 2s            | How long Opensearch waits until the documents are available for search. A positive value defines the duration in seconds. A value of -1 means that a refresh needs to be done manually.                                                                                                                                                                                                                                |
+| opensearch.settings.index.nested_documents_limit | 10000         | Optimize uses nested documents to store list information such as activities or variables belonging to a process instance. This setting defines the maximum number of activities/variables/incidents that a single process instance can contain. This limit helps to prevent out of memory errors and should be used with care. For more information, please refer to the Opensearch documentation on this topic.       |
```

Comment:
> For more information, please refer to the Opensearch documentation on this topic. 

Can we link to this?

Reply:
I don't think it's a good idea because this would be an external link outside of our control. In the elastic equivalent we also don't link to the documentation.